tests/python/test_tools_smoke.py

```diff
@@ -279,6 +279,10 @@ def test_opensnoop(self):
     def test_pidpersec(self):
         self.run_with_int("pidpersec.py")
 
+    @skipUnless(kernel_version_ge(4,17), "requires kernel >= 4.17")
```

Comment:
Please annotate this test with `@mayFail`, currently, we don't have CI for ppc.

Reply:
Okay, will update.

Reply:
patch updated, thanks.